style.css

```diff
@@ -0,0 +1,137 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;700&family=Roboto:wght@700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+}
+
+body {
+  background-image: url(./images/header_bg.svg);
+  background-repeat: no-repeat;
+  background-size: auto;
+  background-position: top 20px left -195px;
+  background-color: #191765;
+  color: #c8c8d4;
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  line-height: 1.75rem;
+  font-size: 1.25rem;
+  max-height: 100vh;
+  width: 100%;
+}
+
+.nav-link {
+  display: none;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 1.5rem;
+  width: 100%;
+}
+
+.header a {
+  color: #fff;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 500;
+  font-size: 1.125rem;
+}
+
+.landing-page {
+  background-image: url(./images/mob-header.svg);
+  background-repeat: no-repeat;
+  background-size: auto;
+  background-position: 290px 440px;
+}
+
+ul {
+  list-style-type: none;
+}
+
+.container {
+  width: 100%;
+  padding: 1.5rem;
+}
+
+.main-intro {
+  text-align: center;
+}
+
+.theme {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 700;
+  font-size: 3rem;
+  color: #7d73fc;
+  letter-spacing: 0.35px;
+  line-height: 4.25rem;
+}
+
+.sub-theme {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 500;
+  font-size: 2rem;
+  line-height: 2.75rem;
+  color: #edecff;
+  margin: 16px 0 50px 0;
+}
+
+.text {
+  margin: 12px 0 40px 0;
+}
+
+.white {
+  border: 1px solid #646369;
+  margin: 40px 0;
+}
+
+.skills-div {
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+  margin: auto;
+  gap: 2rem;
+}
```

Comment:
There is a strange margin to the right of this content that is not present in the template design.

<img width="197" alt="Screen Shot 2022-08-18 at 8 29 57" src="https://user-images.githubusercontent.com/82426682/185407171-693fc691-0add-470e-a1df-3797e38aa38e.png">

Please fix it.